dev/com.ibm.ws.jca/src/com/ibm/ws/jca/internal/WorkProxy.java

```diff
@@ -451,4 +458,11 @@ Long waitForStart() throws InterruptedException, WorkRejectedException {
      * obtain the listener.
      */
     protected WorkListener lsnr;
+
+    //Returns the workContext for this JCA runnable including the hintsContext if it's applicable.
```

Comment:
Please clarify that it returns the Liberty threading context (or whatever you call it).  It does not return a JCA WorkContext.